### PR TITLE
Suspend functionality

### LINKED ISSUE #150: Suspend functionality
Add functionality to suspend registration

### PR DESCRIPTION
**Pull request by mpnogaj**
Fix #150

### CODE REVIEW

File: app/src/conferences/templates/conferences/index.html [hunk 1]
```diff
@@ -20,6 +20,10 @@ <h1 class="header center grey-text text-lighten-4 main_title">
               <h3 class="header center red-text text-lighten-2 main_title ">
                 {% trans "Registration is over" %}
               </h3>
+              {% elif zosia.registration_suspended %}
+              <h3 class="header center red-text text-lighten-2 main_title">
+                {% trans "Registration is suspended"%}
```
Comment (ref-humbold) [inline app/src/conferences/templates/conferences/index.html:25]: I'd add more info, like: `"Registration is suspended, please wait for further information"`.

File: app/src/users/templates/users/profile.html [hunk 2]
```diff
@@ -169,7 +169,9 @@ <h5>
             </h5>
           {% else %}
             <div class="card-content">
-              {% if not registration_open %}
+              {% if zosia.registration_suspended %}
+              <p>{% trans "Registration is suspended"%}</p>
```
Comment (ref-humbold) [inline app/src/users/templates/users/profile.html:173]: I'd add more info, like: "Registration is suspended, please wait for further information".

**Review by Bozydarek — APPROVED** [review]
LGTM :+1:

**Review by ref-humbold — APPROVED** [review]
Good job :)

**Pull request merged**